src/positions/saga.ts

```diff
@@ -258,14 +259,18 @@ export function* handleEnableHooksPreviewDeepLink(
 }
 
 export async function triggerShortcutRequest(hooksApiUrl: string, bodyJson: any) {
+  const { enableAppFee } = getDynamicConfigParams(DynamicConfigs[StatsigDynamicConfigs.SWAP_CONFIG])
   const response = await fetchWithTimeout(
     getHooksApiFunctionUrl(hooksApiUrl, 'triggerShortcut').toString(),
     {
       method: 'POST',
       headers: {
         'Content-Type': 'application/json',
       },
-      body: JSON.stringify(bodyJson),
+      body: JSON.stringify({
+        ...bodyJson,
+        enableSwapFee: enableAppFee,
```

Comment:
This should be specific to swap-deposit. Can we do it earn/prepareTransactions?

Reply:
Fixed in bf7459b287507559c1eed277cf6b1dd7046798ef!